src/main/java/org/webjars/WebJarVersionLocator.java

```diff
@@ -17,70 +18,112 @@ public class WebJarVersionLocator {
     /**
      * The path to where webjar resources live.
      */
-    public static final String WEBJARS_PATH_PREFIX = "META-INF/resources/webjars";
+    public final String WEBJARS_PATH_PREFIX = "META-INF/resources/webjars";
```

Comment:
These could (should) still be static, right? Not sure what the intent was here.

Reply:
I was thinking we may want the opportunity to expose an override (constructor param) at some point in that instance. But maybe that isn't necessary.

Reply:
I would be in favor of keeping these static as well. The uppercase suggest by convention these values are constants  so if we change them that could be confusing. You can maybe introduce non static field later when needed, here that's more to communicate the defaults.